### PR TITLE
Feature/connector poleemploi

### PR DESCRIPTION
**Pull request by nedhirouebnou**

### CODE REVIEW

File: src/hrflow_connectors/connectors/poleemploi/connector.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    lat = poleemploi_location.get("latitude")
+    lat = float(lat) if lat is not None else lat
+
+    lng = poleemploi_location.get("longitude")
+    lng = float(lng) if lng is not None else lng
+
+    concatenate = []
+    for field in ["libelle", "codePostal"]:
+        if poleemploi_location.get(field):
+            concatenate.append(poleemploi_location.get(field))
```
Comment (riquelme222) [inline src/hrflow_connectors/connectors/poleemploi/connector.py:21]: return " ".join([poleemploi_location.get(field) for field in ["libelle", "codePostal"] if poleemploi_location.get(field)])

Always try to use one-liners and compact functions as much as possible 

when you first code code with for loop and test and then refactor your code

File: src/hrflow_connectors/connectors/poleemploi/connector.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        return sections
+    else:
+        sections.append(
+            dict(
+                name="poleemploi_company_description",
+                title="Company Description",
+                description=poleemploi_job["entreprise"]["description"],
+            )
+        )
+    return sections
```
Comment (riquelme222) [inline src/hrflow_connectors/connectors/poleemploi/connector.py:41]: Initialize sections with [] 

Change it only if you need --> add one block of logic 

`    if (
        "entreprise" in poleemploi_job
        and "description" in poleemploi_job["entreprise"]
    ):
                    sections.append(
            dict(
                name="poleemploi_company_description",
                title="Company Description",
                description=poleemploi_job["entreprise"]["description"],
            )
        )
     return sections


`

return sections outside the if block so if you enter it you edit sections if you don't sections is still = [] and you return it either way

File: src/hrflow_connectors/connectors/poleemploi/connector.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+        t("poleemploi_contractType", job.get("typeContratLibelle")),
+        t("poleemploi_experience", job.get("experienceLibelle")),
+        t("poleemploi_salary", salaire.get("libelle")),
+        t("poleemploi_working_hours", job.get("dureeTravailLibelle")),
+        t("poleemploi_qualification", job.get("qualificationLibelle")),
+        t("poleemploi_secteurActivite", job.get("secteurActiviteLibelle")),
+        t("poleemploi_contact-name", contact.get("nom")),
+        t("poleemploi_contact-email", contact.get("courriel")),
+        t("poleemploi_contact-phone", contact.get("telephone")),
+    ]
```
Comment (riquelme222) [inline src/hrflow_connectors/connectors/poleemploi/connector.py:64]: this is good. 

But I'm a fan of not returning big objects it's not readable 

add variable called tags = [list of tags] and return tags it'd be clearer in this case

File: src/hrflow_connectors/connectors/poleemploi/warehouse.py [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+    dureeHebdo: t.Optional[DureeHebdo]
+    dureeHebdoMin: t.Optional[int]  # TODO : format HHMM
+    dureeHebdoMax: t.Optional[int]  # TODO : format HHMM
+    dureeContratMin: t.Optional[
+        float
+    ]  # TODO : Décimal positif (Séparateur de décimales: '.')
+    dureeContratMax: t.Optional[
+        float
+    ]  # TODO : Décimal positif (Séparateur de décimales: '.')
+    offresManqueCandidats: t.Optional[bool]
```
Comment (riquelme222) [inline src/hrflow_connectors/connectors/poleemploi/warehouse.py:193]: @nedhirouebnou What is this TODO ? 
Why don't you do it ? 

Use TODO tree extension go over all your code and fix all them TODOs

File: src/hrflow_connectors/connectors/poleemploi/warehouse.py [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+    # commune : t.Optional[Commune]
+    distance = (
+        10  # distance=0 pour pour obtenir seulement les offres d'une commune spécifique
+    )
+    departement: t.Optional[str]
+    inclureLimitrophes: t.Optional[bool]
+    # region : t.Optional[Region]
+    # paysContinent : t.Optional[PaysContinent]
+    # niveauFormation : t.Optional[NiveauFormation]
+    # permis : t.Optional[Permis]
```
Comment (riquelme222) [inline src/hrflow_connectors/connectors/poleemploi/warehouse.py]: @nedhirouebnou The commented lines group them and put them at the end. Try to add a comment explaining the need to get the list of values from an endpoint etc ... be explicit, so the next person working on your code can understand why you left them commented, or even you in a few months you won't be able to remember why you did this 
ZEN of PYTHON : "Explicit is better than Implicit"

Comment (nedhirouebnou) [reply src/hrflow_connectors/connectors/poleemploi/warehouse.py]: These are the fields for which I implemented the references in the later commit.

File: src/hrflow_connectors/connectors/poleemploi/schemas.py [hunk 6]
```diff
@@ -0,0 +1,277 @@
...
+class Langue(BaseModel):
+    libelle: str = Field(
+        description=(
+            "Language label"
+            "A GET request for the list of accepted choices from the Offres"
+            " d'emploi API"
+            "to this endpoint :"
+            " {POLEEMPLOI_REFERENCES_ENDPOINT}referentiel/langues"
+        ),
+    )
```
[the remark this answers is not in the document]
Comment (nedhirouebnou) [reply src/hrflow_connectors/connectors/poleemploi/schemas.py:96]: fixed

**Pull request closed without merging**